.gitignore

```diff
@@ -160,4 +160,4 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
```

Comment:
Any particular reason for this?

Reply:
Emm, I use PyCharm to write all the codes and try to ignore this folder which is created by JetBrains.